src/main/java/org/terasology/signalling/components/SignalConsumerAdvancedStatusComponent.java

```diff
@@ -20,6 +20,10 @@
 import java.util.HashMap;
 import java.util.Map;
 
+/**
+ * Keeps track of the signal strength on each side of an entity
+ * -1 is infinite signal, 0 is no signal
+ */
```

Comment:
The information on the fields should be documented with the fields.
So the information on the negative being infinite should be on the field itself rather than on the class